integrations/nifi/nifi-nar-bundles/nifi-standard-services/nifi-spark-service-bundle/nifi-spark-service/src/test/java/com/thinkbiganalytics/nifi/v2/spark/SparkJobserverServiceIT.java

```diff
@@ -0,0 +1,251 @@
+import org.junit.Test;
+
+import java.util.List;
+
+public class SparkJobserverServiceIT {
+
+    /**
+     * Identifier for the spark jobserver service
+     */
+    private static final String SPARK_JOBSERVER_SERVICE_IDENTIFIER = "sparkJobServerService";
+
+    /**
+     * Processor property for the cleanup event service
+     */
+    private static final PropertyDescriptor SPARK_JOBSERVER_SERVICE_PROPERTY = new PropertyDescriptor.Builder()
+        .name("Spark Jobserver Service")
+        .description("Provides long running spark contexts and shared RDDs using Spark jobserver.")
+        .identifiesControllerService(SparkJobserverService.class)
+        .required(true)
+        .build();
+
+    /**
+     * Spark Jobserver service for testing
+     */
+    private static final SparkJobserverService sparkJobserverService = new SparkJobserverService();
+    private static final String sparkJobserverUrl = "http://localhost:8089";
+    private static final String syncTimeout = "600";
+
+    /**
+     * Default Context Creation Properties
+     */
+    private static final String numExecutors = "1";
+    private static final String memPerNode = "512m";
+    private static final String numCPUCores = "2";
+    private static final SparkContextType sparkContextType = SparkContextType.SPARK_CONTEXT;
+    private static final int contextTimeout = 0;
+    private static boolean async = false;
+
+    /**
+     * Default Execute Spark Context Job Properties
+     */
+    private static final String appName = "test-app";
+    private static final String classPath = "spark.jobserver.WordCountExample";
+    private static final String args = "input.string = a b c a b see";
+
+    /**
+     * Test runner
+     */
+    private final TestRunner runner = TestRunners.newTestRunner(new AbstractProcessor() {
+        @Override
+        protected List<PropertyDescriptor> getSupportedPropertyDescriptors() {
+            return ImmutableList.of(SPARK_JOBSERVER_SERVICE_PROPERTY);
+        }
+
+        @Override
+        public void onTrigger(ProcessContext context, ProcessSession session) throws ProcessException {
+            // do nothing
+        }
+    });
+
+    /**
+     * Initialize instance variables.
+     */
+    @Before
+    public void setUp() throws Exception {
+
+        // Setup test runner
+        runner.addControllerService(SPARK_JOBSERVER_SERVICE_IDENTIFIER, sparkJobserverService);
+        runner.setProperty(SPARK_JOBSERVER_SERVICE_PROPERTY, SPARK_JOBSERVER_SERVICE_IDENTIFIER);
+        runner.setProperty(sparkJobserverService, SparkJobserverService.JOBSERVER_URL, sparkJobserverUrl);
+        runner.setProperty(sparkJobserverService, SparkJobserverService.SYNC_TIMEOUT, syncTimeout);
+        runner.enableControllerService(sparkJobserverService);
+    }
+
+    /**
+     * Verify creating Spark Context and then deleting the Context
+     */
+    @Test
+    @Ignore
+    public void testContextCreationAndDeletion() throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Fix or remove this skipped unit test [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1607)